src/server/document-controller.js

```diff
@@ -290,6 +301,12 @@ class DocumentController {
       '/api/v1/docs/tags/*',
       (req, res) => this.handleGetDocTags(req, res)
     );
+
+    router.post(
+      '/api/v1/docs/regenerate-documents-batch',
```

Comment:
To me this looks a bit misplaced and also not inline with the existing batch creation.

What about extracting this in its separate controller and having the endpoint be
`POST /api/v1/document-regenerations/`?
Equivalent to imports.

Reply:
When I put the endpoint here I looked the imports and the batch belongs to the import. Analogously, the regenerate documents batch belongs to the docs (or so I thought).

Reply:
We could leave it here for now and have a ticket to unify batch processing:

* Having a batch controller
* Having a batch service
* Make the UI show batches of all kinds (imports and regenerations)

Reply:
I've created the story here: https://educandu.atlassian.net/browse/EDU-250